src/content/pt/2019/compression.md

```diff
@@ -0,0 +1,369 @@
+---
+#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
+title: Compressão
+description: Capítulo de compressão do Web Almanac de 2019 acerca de compressão HTTP, algoritmos, tipos de conteúdo, compressão própria ou por terceiros e oportunidades.
+authors: [paulcalvano]
+reviewers: [obto, yoavweiss]
+analysts: [paulcalvano]
+editors: [bazzadp]
+translators: [soulcorrosion]
+discuss: 1770
+results: https://docs.google.com/spreadsheets/d/1IK9kaScQr_sJUwZnWMiJcmHEYJV292C9DwCfXH6a50o/
+paulcalvano_bio: Paul Calvano é um arquitecto de Performance Web na <a href="https://www.akamai.com/">Akamai</a>, onde ajuda empresas a melhorar o desempenho dos seus sites. Ele é também umas das pessoas que mantém o projecto HTTP Archive. Podem encontrá-lo no Twitter em <a href="https://twitter.com/paulcalvano">@paulcalvano</a>, a escrever no seu blog <a href="https://paulcalvano.com">http://paulcalvano.com</a> e a partilhar pesquisas do HTTP Archive em <a href="https://discuss.httparchive.org">https://discuss.httparchive.org</a>.
+featured_quote: A compressão HTTP é uma técnica que permite codificar informação usando menos bits que a represantação original. Quando é usado para entrega de conteúdo web, isso permite a servidores web reduzir a quantidade de dados transmitodos aos clientes. Isto aumenta a eficiência da largura de banda disponível do cliente, reduz o peso da página e melhora o seu desempenho.
+featured_stat_1: 38%
+featured_stat_label_1: Respostas HTTP que usam compressão baseada em texto
+featured_stat_2: 80%
+featured_stat_label_2: Uso da compressão Gzip
+featured_stat_3: 56%
+featured_stat_label_3: Respostas HTML sem compressão
+---
+
+## Introdução
+
+A compressão HTTP é uma técnica que permite codificar informação usando menos bits que a representação original. Quando é usado para entrega de conteúdo web, isso permite a servidores web reduzir a quantidade de dados transmitidos aos clientes. Isto aumenta a eficiência da largura de banda disponível do cliente, reduz o [peso da página](./page-weight) e melhora o seu [desempenho](./performance).
+
+Algoritmos de compressão são muitas vezes categorizados como podendo ter ou não perdas:
+
+* Quando um algoritmo de compressão com perdas é usado, o processo é irreversível e o ficheiro original não pode ser restaurado através de descompressão. É uma prática comum para comprimir recursos media, tal como imagens e vídeo onde perder alguns dados não irá materialmente afectar o recurso.
+* A compressão sem perdas é um processo totalmente reversível sendo, tipicamente, usado em recursos do tipo texto, como [HTML](./markup), [JavaScript](./javascript), [CSS](./css), etc.
+
+Neste capítulo, vamos explorar como conteúdo de texto é comprimido na web. Análise de conteúdo não baseado em texto faz parte do capítulo [Media](./media)
+
```

Comment:
We want to keep the line numbers the same as English, so non-Portugues speakers can still make technical corrections if needed (e.g. if a link was wrong) so can we remove this extra line? I only cleaned these up recently in #1998 so probably you started on this translation before then.

```suggestion
```